src/main/tut/categories-great-and-small.md

```diff
@@ -0,0 +1,26 @@
+```Haskell
+class Monoid m where
+    mempty  :: m
+    mappend :: m -> m -> m
+```
+```scala
+trait Monoid[M] {
+  def mempty: M
+  def mappend(m1: M)(m2: M): M
+}
+```
+................
+```Haskell
+instance Monoid String where
+    mempty = ""
+    mappend = (++)
+```
+```scala
+object Monoid {
+  implicit object stringMonoid extends Monoid[String] {
```

Comment:
I'd prefer `implicit def stringMonoid: Monoid[String] = new Monoid[String]`, using objects in this way can sometimes give you implicit resolution problems due to overly specific types being inferred. Given that this will end up in a fairly authoritative source, I'd rather play safe